FlowersStore/Scenes/FlowersList/FlowerItemCell.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  FlowerItemCell.swift
+//  FlowersStore
+//
+//  Created by Renato on 03/11/21.
+//
+
+import Foundation
+import UIKit
+
+struct Koe {
+    let name: String
+}
+
+protocol FlowerItemCellDelegate: AnyObject {
+    func didTapFavoriteButton(_ cell: FlowerItemCell, flower: Flower)
```

Comment:
<!--
  0 failure: 
  2 warning:  Parameter 'cell' ..., Parameter 'flower...
  
  
  DangerID: danger-id-Danger;
  File: FlowersStore/Scenes/FlowersList/FlowerItemCell.swift;
  Line: 16;
-->

- :warning: Parameter 'cell' is unused
- :warning: Parameter 'flower' is unused